config/config.ini

```diff
@@ -79,3 +79,10 @@ TransactionTax = 0.15
 [Betting]
 RollCases = 66, 90, 99, 100
 RollReturns = 0, 2, 4, 10
+
+[Roles]
+Pronouns = She/Her, He/Him, They/Them
+Fields = Anatomy and Cell Biology, Anthropology, Agriculture, Architecture, Biochemistry, Bioengineering, Biology, Bioresource Engineering, Chemical Engineering, Chemistry, Civil Engineering, Classics, cogito, Commerce, Computer Engineering, Computer Science, Computer Science/Biology, Cultural Studies, Desautels, Economics , Electrical Engineering, English, Experimental Medicine, Finance, Geography, History, Indigenous Studies, International Development Studies, Jewish Studies, mac kid, Materials Engineering, Math, MBA, Mechanical Engineering, Medicine, Microbiology and Immunology, Neuroscience, Nursing, Pharmacology, Philosophy, Physical Therapy, Physics, Physiology, Political Science, Psychiatry, Psychology, Public Health, Social Work, Sociology, Software Engineering, Statistics
+Faculties = Science, Engineering, Management, Arts, ArtSci, Agriculture and Environment, Law, Education, Dentistry, Music
+Years = U0, U1, U2, U3, U4, PhD, Masters, Post Doc, wenthere
+Generics = weeb, weeb stomper
```

Comment:
This is just the default suggested set but the set used on our server can obviously differ from this.